include/manifold/manifold.h

```diff
@@ -46,11 +46,59 @@ class CsgLeafNode;
  * @brief Mesh input/output suitable for pushing directly into graphics
  * libraries.
  *
- * This may not be manifold since the verts are duplicated along property
- * boundaries that do not match. The additional merge vectors store this missing
- * information, allowing the manifold to be reconstructed. MeshGL is an alias
- * for the standard single-precision version. Use MeshGL64 to output the full
- * double precision that Manifold uses internally.
+ * The core (non-optional) parts of MeshGL are the triVerts indices buffer and
+ * the vertProperties interleaved vertex buffer, which follow the conventions of
```

Comment:
Maybe spell them out? E.g. `triVerts[3 * i], triVerts[3 * i + 1], triVerts[3 * i + 2]` are the three vertices of a triangle, etc. So users not familiar with graphics library don't have to look them up.